README.md

```diff
@@ -60,17 +56,13 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 [Its Me Again] <a name="about-project">Hello Microverse</a>
 
-## 🛠 Built With <a name="built-with"></a>
+**[Hello Microverse]** is a practice project aimed at sharpening programming knowledge of beginner programmers.
 
-### Tech Stack <a name="tech-stack"></a>
+## 🛠 Built With <a name="built-with">html and css</a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack <a name="tech-stack">PERN</a>
 
 <details>
```

Comment:
- [ ] I kindly suggest that you customize this section so that it contains information which are related to your current project. 

## EXAMPLE

```.md
### Tech Stack <a name="tech-stack"></a>

<details>
  <summary>Client</summary>
  <ul>
    <li><a href="https://reactjs.org/](https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
  </ul>
</details>

```